src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -63,8 +37,6 @@ export const AllEarnOpportunities = () => {
     dispatch,
   } = useWallet()
 
-  const { SaversVaults } = useSelector(selectFeatureFlags)
```

Comment:
This now lives at resolver-level, the whole `saversVault` thing was a hack for the view-layer work to be done in isolation

Reply:
tiny issue - if we turn off flag the cached savers opportunities still appear, but this is a ways away from being enabled in prod so i'm not concerned with it for this PR